wandb/integration/keras/keras.py

```diff
@@ -64,6 +47,10 @@ def is_generator_like(data):
         data, types))
 
 
+def patch_keras():
+    # TODO
+    pass
```

Comment:
What's intended here?  I believe regular keras doesn't need to be patched.

Reply:
Will remove after testing against regular keras.

Reply:
Nope. Getting `wandb: WARNING No validation_data set, pass a generator to the callback.` with regular keras. Will take a look tomorrow.

Reply:
Fixed